Dockerfile

```diff
@@ -1,19 +1,18 @@
 FROM golang:1.12-alpine AS build
 RUN apk add --no-cache git
+WORKDIR /tmp
+RUN wget https://github.com/buildpack/pack/releases/download/v0.3.0/pack-v0.3.0-linux.tgz
+RUN tar -xzf pack-v0.3.0-linux.tgz
+RUN wget https://raw.githubusercontent.com/buildpack/pack/v0.3.0/LICENSE
```

Comment:
These 3 `RUN` can also be packed as you did below, something like:

```
RUN wget https://github.com/buildpack/pack/releases/download/v0.3.0/pack-v0.3.0-linux.tgz && \
       tar -xzf pack-v0.3.0-linux.tgz && \
       wget https://raw.githubusercontent.com/buildpack/pack/v0.3.0/LICENSE
```

Reply:
Yeah, that works, but since this is an intermediate image, we don't have to squash layers too aggressively. I think maximizing cache potential is better in this case.